go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/golang/mock v1.3.1
 	github.com/golang/protobuf v1.3.2
 	github.com/golang/snappy v0.0.0-20170215233205-553a64147049
+	github.com/google/btree v1.0.0 // indirect
```

Comment:
Do these new entries persist after `go mod tidy`? I don't quite understand what's happening, but I've noticed the `go` tool adding some things that we don't actually need.

Reply:
I hadn't intended to commit a new `go.mod` :(
But this one diff does persist after `go mod tidy`